packages/ledgerwallet-tezos/ledgerwallet-tezos.0.1.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+name: "ledgerwallet-tezos"
+authors: "Vincent Bernardoff <vb@luminar.eu.org>"
+maintainer: "Vincent Bernardoff <vb@luminar.eu.org>"
+license: "ISC"
+homepage: "https://github.com/vbmithr/ocaml-ledger-wallet"
+bug-reports: "https://github.com/vbmithr/ocaml-ledger-wallet/issues"
+dev-repo: "git+https://github.com/vbmithr/ocaml-ledger-wallet"
+doc: "https://vbmithr.github.io/ocaml-ledger-wallet/doc"
+build:    [ "dune" "build"   "-p" name "-j" jobs ]
+# run-test: [ "dune" "runtest" "-p" name "-j" jobs ]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "2.4.0"}
+  "ledgerwallet"
```

Comment:
Would it be ok to sync those two packages and add a `{= version}` constraint here?